src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -42,6 +46,12 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
+
```

Comment:
> Complex is better than complicated

This can be simplified by reusing your `getNumberOfOccurrences` method.

```java
return getNumberOfOccurrences(array, value) > 0;
```